### ISSUE TITLE
Tailwind Styles for UI not Applying for Vitejs React Application

### ISSUE THREAD

**Issue by Aliemeka** [vercel/turborepo#722]
### What version of Turborepo are you using?

latest

### What package manager are you using / does the bug impact?

Yarn v1

### What operating system are you using?

Mac

### Describe the Bug

Anytime I import components from UI, the tailwind styles are not applying on the app level.

ui in packages already has styles built in to it's tailwind.config.js file

But anytime I run the application I don't see the styles

There is a tailwind config at the app level

### UI package.json
`
{
  "name": "ui",
  "version": "0.0.0",
  "main": "./index.tsx",
  "types": "./index.tsx",
  "license": "UNLICENSED",
  "devDependencies": {
    "@types/react": "^17.0.37",
    "@types/react-dom": "^17.0.11",
    "@types/jest": "^26.0.22",
    "autoprefixer": "^10.4.2",
    "config": "*",
    "eslint": "^7.32.0",
    "jest": "^26.6.3",
    "postcss": "^8.4.6",
    "tailwindcss": "^3.0.19",
    "tsconfig": "*",
    "typescript": "^4.5.3"
  }
}
`

### Expected Behavior

Applied tailwind styles should be applied for the shared UI components

### To Reproduce

Create react app with vitejs for app

Create ui components with react and configure tailwind

**Comment by Aliemeka** [comment]
Only happens to work if I copy the components folder from ui and paste it in the app directory. Then it seems to work even when I delete it afterwards


But from the start it does

**Comment by Aliemeka** [comment]
Figured adding ui to the purge/content scope solved the issue

**Comment by salahqasm1** [comment]
@Aliemeka where did you put it exactly? 
i saw a solution where i have to put in apps/appName/tailwind.config.js in content
"../packages/ui/src/**/*.tsx"
which i dont think it is the right way to do it

**Comment by BadMachine** [comment]
Having the same problem with setting up Turborepo based on Vite + React.
Once I moved shared components to /packages/ui the styles not applied to to components anymore.
I spent around 8 hours, still no luck.
I feel like its the bundler problem: it builds shared components first and then inserts it in APPS, by that time inserted components have their classnames, but application didnt know it should include the relevant classes in the final bundle from Tailwind.
